base/cvd/cuttlefish/host/commands/cvd/server_command/stop.cpp

```diff
@@ -133,31 +131,28 @@ Result<selector::LocalInstanceGroup> CvdStopCommandHandler::PromptUserForGroup(
                               }),
                groups.end());
   GroupSelector selector{.groups = groups};
+  request.Out() << selector.Menu() << "\n";
   auto menu = selector.Menu();
 
-  CF_EXPECT_EQ(WriteAll(request.Out(), menu + "\n"), (ssize_t)menu.size() + 1);
   std::unique_ptr<InterruptibleTerminal> terminal_ =
-      std::make_unique<InterruptibleTerminal>(request.In());
+      std::make_unique<InterruptibleTerminal>();
 
-  const bool is_tty = request.Err()->IsOpen() && request.Err()->IsATTY();
+  const bool is_tty = isatty(2);
```

Comment:
This should check `0` and `1`, since it uses `request.Out()` and `request.In()` below.

Given that the check is performed on those file descriptors, the code below should use `std::cin` and `std::cout` instead of `request.In()` and `request.Out()`.

Reply:
Never mind, this check is on `2` because it prints with colors later on `cerr`